src/server/chromium/videoRecorder.ts

```diff
@@ -53,7 +53,7 @@ export class VideoRecorder {
     assert(!this._isRunning());
     const w = options.width;
     const h = options.height;
-    const args = `-loglevel error -f image2pipe -c:v mjpeg -i - -y -an -r ${fps} -c:v vp8 -qmin 0 -qmax 50 -crf 8 -vf pad=${w}:${h}:0:0:gray,crop=${w}:${h}:0:0`.split(' ');
+    const args = `-loglevel error -f image2pipe -c:v mjpeg -i - -y -an -r ${fps} -c:v vp8 -minrate 800K -maxrate 1M -b:v 1M -vf pad=${w}:${h}:0:0:gray,crop=${w}:${h}:0:0`.split(' ');
```

Comment:
Won't `-minrate 800K` result in a much bigger file with big resolution video?

Reply:
dunno!